src/app-layout/visual-refresh/header.tsx

```diff
@@ -18,7 +18,6 @@ export default function Header() {
       className={clsx(
         styles.content,
         {
-          [styles['has-breadcrumbs']]: breadcrumbs,
```

Comment:
`.has-breadcrumbs` class does not exist anymore, so it renders `<header className="undefined">`